client/src/Components/Template/Template.jsx

```diff
@@ -1,21 +1,21 @@
 import React from 'react';
-import { Container } from '@material-ui/core';
+import { Container, Box } from '@material-ui/core';
 
 import Header from 'Components/Header/Header';
-import Footer from 'Components/Footer/Footer';
+// import Footer from 'Components/Footer/Footer';

```

Comment:
nie zostawiaj wykomentowanych linii. mysle ze mozesz to narazie wywalic, ten footer nawet nie jest w planach